### ISSUE TITLE
/admin, /console  not comnecting to console session 

### ISSUE THREAD

**Issue by bernd-wechner** [FreeRDP/FreeRDP#6588]
I was looking at a problem described here:

https://superuser.com/questions/823927/software-does-not-open-over-a-vpn-rdp-connection

and there was the suggestion that an RDP connection could be requested to the Windows console rather than to terminal services.

QWINSTA is the tools I'd use on the host to check the session, and can see when I connect that it's an RDP session:

```
PS E:\> QWINSTA
 SESSIONNAME       USERNAME                 ID  STATE   TYPE        DEVICE
 services                                    0  Disc
>rdp-tcp#13        berndw                    2  Active
 console                                     4  Conn
 rdp-tcp                                 65537  Listen
```

The suggestion left on superuser.com (admittedly in 2014) is to connect using the MS client:

`MSTSC /v:RemoteComputerName /admin`

Which I can't do as I'm on Linux at the client side using xfreerdp (which is rocking awesome BTW, thanks heaps, it's works like a dream) so I checked the man page and:

```
       /admin, /console
           Admin (or console) session
```

which looks hot!  But alas, doesn't work, if I connect with this and run QWINSTA on the host, I'm still on a Terminal Services session (though the trailing digit in the session ID goes up one).

When I run xfreerdp on my client side in bash, it produces a trace but not clue as to whether or not it even noticed my /admin or /console request:

```
[11:33:22:354] [586277:586278] [INFO][com.freerdp.core] - freerdp_connect:freerdp_set_last_error_ex resetting error state
[11:33:22:354] [586277:586278] [INFO][com.freerdp.client.common.cmdline] - loading channelEx rdpdr
[11:33:22:354] [586277:586278] [INFO][com.freerdp.client.common.cmdline] - loading channelEx rdpsnd
[11:33:22:354] [586277:586278] [INFO][com.freerdp.client.common.cmdline] - loading channelEx cliprdr
[11:33:22:674] [586277:586278] [INFO][com.freerdp.primitives] - primitives autodetect, using optimized
[11:33:22:676] [586277:586278] [INFO][com.freerdp.core] - freerdp_tcp_is_hostname_resolvable:freerdp_set_last_error_ex resetting error state
[11:33:22:676] [586277:586278] [INFO][com.freerdp.core] - freerdp_tcp_connect:freerdp_set_last_error_ex resetting error state
[11:33:22:794] [586277:586278] [WARN][com.freerdp.crypto] - Certificate verification failure 'unable to get local issuer certificate (20)' at stack position 0
[11:33:22:794] [586277:586278] [WARN][com.freerdp.crypto] - CN = redacted.com.au
[11:33:24:998] [586277:586278] [INFO][com.freerdp.gdi] - Local framebuffer format  PIXEL_FORMAT_BGRX32
[11:33:24:998] [586277:586278] [INFO][com.freerdp.gdi] - Remote framebuffer format PIXEL_FORMAT_RGB16
[11:33:24:054] [586277:586278] [INFO][com.winpr.clipboard] - initialized POSIX local file subsystem
[11:33:24:067] [586277:586278] [INFO][com.freerdp.channels.rdpsnd.client] - [static] Loaded fake backend for rdpsnd
[11:33:25:770] [586277:586278] [INFO][com.freerdp.core] - rdp_set_error_info:freerdp_set_last_error_ex resetting error state
```

After the last line it opens up the session and I'm rolling. 

I guess if there's a bug here it i one of:

1. The man page is not clear on what that /admin /console options do and really should be more descriptive.
2. If it's supposed to work like the same option on MSTSC, well apparently it's not (I admit I don't have MSTSC on hand to confirm that it does ;-). And I'll also admit that the MS manp page for it is equally light on as the xfreerdp one, saying only `/admin | Connects you to a session for administering the server.`

In other words, at the very least clarity could improve here and in the worst case there's a bug in the actual /admin /console implementation.

**Comment by akallabeth** [comment]
I can understand your issue, but the options are flags in the RDP protocol that have quite a light description.
Depends a lot on the RDP server in use what these actually do.
Basically this option only sets a flag so that the remote session is a `admin` session (whatever that means for the RDP server)

**Comment by bernd-wechner** [comment]
Cool. Pinged my admin to look for server configs. If anyone has any tips on server configs that have the RDP (Terminal Services) service (Is that a bit like PIN number and ATM machine?) work on a Console session on a Terminal Services session.

**Comment by akallabeth** [comment]
I assume the issue is resolved with this?
Closing, repoen/ping me if not.
